module/LearnZF2Captcha/config/captcha.config.php

```diff
@@ -0,0 +1,37 @@
+<?php
+/**
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+ * "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT
+ * LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR
+ * A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT
+ * OWNER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL,
+ * SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT
+ * LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE,
+ * DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY
+ * THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+ * (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+ * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ *
+ * This software consists of voluntary contributions made by many individuals
+ * and is licensed under the MIT license.
+ */
+/**
+ * Captcha config.
+ *
+ * @author Abdul Malik Ikhsan <samsonasik@gmail.com>
+ */
+return [
+    'learnzf2_captcha_config' => [
+        0 => [
+            'adapter_name' => 'Dumb',
+            'options' => [],
+        ],
+        1 => [
+            'adapter_name' => 'Image',
+            'options' => [
+                'width' => 200,
+                'height' => 100,
+            ],
+        ],
```

Comment:
I am thinking to add recaptcha adapter, but it will take more step for user to get it works. So I only want to include 2 of the adapters. any idea ?